server/src/schema.graphql

```diff
@@ -25,9 +25,15 @@ type SettingsContracts {
   nftFaucet: String!
 }
 
+type SettingsAdmin {
+  address: String!
+  secret: String!
+}
+
 type Settings {
   tzStatsUrl: String!
   rpc: String!
+  admin: SettingsAdmin!
```

Comment:
why do you need admin settings on the client side?
You are using wallet signer now.